generated/gtkd/atk/ActionIF.d

```diff
@@ -27,7 +27,6 @@ module atk.ActionIF;
 private import atk.c.functions;
 public  import atk.c.types;
 private import glib.Str;
-public  import gtkc.atktypes;
```

Comment:
The makefile usually adds these line by passing `--use-bind-dir` to girtod for backwards compatibility.